hudi-spark-datasource/hudi-spark3-common/src/main/scala/org/apache/spark/sql/hudi/analysis/HoodieSpark3Analysis.scala

```diff
@@ -225,6 +224,15 @@ case class HoodieSpark3ResolveReferences(spark: SparkSession) extends Rule[Logic
         case (u: UnresolvedFieldName, prop) => resolveFieldNames(cmd.table, u.name, u) -> prop
         case other => other
       })
+
+    // Convert to DescribeTableCommand
+    case dt @ DescribeTable(plan @ ResolvesToTable(table), formatted, extended, output) if dt.resolved  =>
```

Comment:
Another approach is to add post analysis rule to convert `DescribeTableCommand` to `DescribeHoodieTableCommand` in `HoodiePostAnalysisRule` (in `HoodieAnalysis.scala`), like converting `CreateDataSourceTableCommand` to `CreateHoodieTableCommand`.  Would that be feasible and easier?

Reply:
Of course it's easier. Didn't know that, thanks for this clue. I'll fix it